pom.xml

```diff
@@ -20,7 +20,7 @@
 
     <groupId>com.ibm.websphere.azure</groupId>
     <artifactId>azure.websphere-traditional.cluster</artifactId>
-    <version>1.3.4</version>
+    <version>1.3.5</version>
```

Comment:
Ignore the version bumping here as it's required by upgrading the test Azure application offer, which is internal and not visible to user.